lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -135,6 +135,15 @@ def build_client(sms_config)
         )
       end
 
+      def origination_number(country_code, sms_config)
+        if !PinpointSupportedCountries::SENDER_ID_COUNTRIES.include?(country_code) &&
```

Comment:
We use a different way to get this config elsewhere in the file, I think it's more consistent this way?
```suggestion
        if !Telephony.config.country_sender_ids.key?(country_code.to_s) &&
```

Reply:
Whoops, I meant to drop the sender ID check (I've removed in the following commit)